data/Makefile.am

```diff
@@ -13,7 +13,7 @@ autostart_DATA = com.endlessm.GoogleChromeInitialSetup.desktop
 com.endlessm.GoogleChromeInitialSetup.desktop: com.endlessm.GoogleChromeInitialSetup.desktop.in Makefile
 	$(AM_V_GEN) sed -e "s|\@pkglibexecdir\@|$(pkglibexecdir)|" $<> $@
 
-gsettings_SCHEMAS = com.endlessm.GoogleChromeInitialSetup.gschema.xml
+gsettings_SCHEMAS = com.endlessm.GoogleChromeHelper.gschema.xml
```

Comment:
You changed the name of the file referenced here but did not rename the actual file, please do the later too.